site/src/pages/release-notes/1.16.1.mdx

```diff
@@ -0,0 +1,16 @@
+---
+date: 2022-12-06
+---
+
+## 🛠️ Bug fixes
+
+- You can now use the exception, which is raised in the decorator, to make a decision for retrying. #4558 #4559
```

Comment:
I think this can be simplified

```suggestion
- You can now use the exception raised in a decorator to make a decision for retrying. #4558 #4559
```